.gitignore

```diff
@@ -8,3 +8,5 @@ build*/
 help/*.qhc
 3rd-party/*/download
 !3rd-party/*/download/README.txt
+.idea/
+cmake-build-debug/
```

Comment:
These lines cannot be merged. Note that directories named build*/ are already ignored.